packages/commerce-events-sdk/README.md

```diff
@@ -36,6 +36,17 @@ To install the script as a dependency, run this command.
 npm install @adobe/commerce-events-sdk
 ```
 
+### Custom Data Layer name
+
+When you are using a customDataLayer name for the Adobe Client Data Layer package, you can pass in the name to the script using the data attribute `data-layer-name`:
+
+```html
+<script>
+    window.myDataLayer = window.adobeDataLayer || [];
+</script>
+<script src="https://unpkg.com/@adobe/commerce-events-sdk/dist/index.js" data-layer-name="myDataLayer"></script>
+```
+
```

Comment:
is this the right convention for script param names? no camelcase (`dataLayerName`)?